core/imageroot/usr/local/nethserver/agent/actions/get-status/validate-input.json

```
@@ -0,0 +1,10 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "get-status input",
+    "$id": "http://schema.nethserver.org/module/get-status-input.json",
```

URL space for inherited actions:

```suggestion
    "$id": "http://schema.nethserver.org/agent/get-status-input.json",
```